.github/workflows/gofmt.yml

```diff
@@ -10,6 +10,6 @@ jobs:
       - uses: actions/checkout@main
       - uses: actions/setup-go@main
         with:
-          go-version: 1.19
+          go-version: "1.20"
```

Comment:
It might be nice to use the `v4` version of setup-go so the builds can reference the mod file and the actions need not be updated. Snippet below
```
      uses: actions/setup-go@v4
      with:
        go-version-file: go.mod
```

Reply:
I have filed - https://github.com/sustainable-computing-io/kepler-operator/issues/101